froide/foirequest/views/make_request.py

```diff
@@ -132,14 +135,38 @@ def get_user_template_vars(self):
         return user_vars
 
     def get_js_context(self):
+        skip_intro_howto = False
+        show_skip_intro_howto_preference = False
+        if self.request.user.is_authenticated:
+            show_skip_intro_howto_preference = True
+            preferences = get_preferences_for_user(
+                self.request.user, [make_request_intro_skip_howto_pref]
+            )
+            if (make_request_intro_skip_howto_pref.key in preferences) and preferences[
+                make_request_intro_skip_howto_pref.key
+            ].value:
+                skip_intro_howto = True
+            elif FoiRequest.objects.filter(user=self.request.user).count() > 50:
```

Comment:
Should be sufficient, I think.

```suggestion
            elif FoiRequest.objects.filter(user=self.request.user).count() > 5:
```